ukrainian/take_almost_all/take_almost_all.md

```diff
@@ -0,0 +1 @@
+[Беру з собою мотузку, запальничку, ніж та виделку і стрибаю через стіну!](https://github.com/udacity/create-your-own-adventure/blob/master/ukrainian/break-through-wall/break-through-wall.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.